### PR TITLE
finalize strings and styles

### LINKED ISSUE #136: Review all strings
@groovecoder have we done this yet? If not, I'll kick off the process today. Shouldn't be too hard since there's really not very many strings in the add-on

### LINKED ISSUE #139: UI style nits
Going to do one last polish pass on the new UI to pick off some very minor details.

### PR DESCRIPTION
**Pull request by johngruen**
fixes #136 and fixes #139

### CODE REVIEW

File: html/popup.html [hunk 1]
```diff
@@ -12,16 +12,16 @@
     <div id="main-panel">
     <div class="row">
       <div class="columns">
-        <div class="blocking title">Blok is blocking tracker requests on this site.</div>
-        <div class="disabled title hide">Parts of this site may be tracking your activity.</div>
+        <h5 class="blocking title">Tracking Protection is enabled.<br> It is blocking tracker requests on this site.</h5>
+        <h5 class="disabled title hide">Tracking Protection is Disabled.<br> Parts of this site may be tracking your activity.</>
```
Comment (pdehaan) [inline html/popup.html]: This seems weird because "enabled" is lowercase (above) and "Disabled" isn't.

**Comment by johngruen** [comment]
@groovecoder not sure why this is failing exactly, take a look for me

File: html/popup.html [hunk 2]
```diff
@@ -75,11 +73,9 @@
             </fieldset>
           </div>
           <div class="row">
-            <label class="notes-label">
-              Why do you think Blok caused this problem?
+            <label class="notes-label">Why do you think Tracking Protection caused this problem?</label>
```
Comment (pdehaan) [inline html/popup.html]: I'm not good at a11y, but not sure if you need to say what tag this `<label>` is for, now that you're not nesting the textarea inside it.

File: js/popup.js [hunk 3]
```diff
@@ -24,12 +24,30 @@ function showFeedbackPanel () {
   show('#feedback-panel')
 }
 
+// grabbed from http://stackoverflow.com/questions/13203518/javascript-date-suffix-formatting
+// for clean date formatting
+// TODO: find an alternate solution if we ever L10N
```
Comment (pdehaan) [inline js/popup.js]: Fwiw, **moment** works well w/ l10n and formatting http://momentjs.com/docs/#/displaying/format/

File: js/popup.js [hunk 4]
```diff
@@ -24,12 +24,30 @@ function showFeedbackPanel () {
...
+    }
+  }
+  return 'th'
+}
+
 function showHostReport (hostReport) {
   let date = new Date(hostReport.dateTime)
-  let hostReportDateTimeString = days[date.getDay()] + ', ' + months[date.getMonth()] + ' ' + date.getDate()
-  document.querySelector('.host-report-date').innerText = ' on ' + hostReportDateTimeString
+  let hostReportDateTimeString = days[date.getDay()] + ', ' + months[date.getMonth()] + ' ' + date.getDate() + ordinal(date.getDate())
```
Comment (pdehaan) [inline js/popup.js]: We could probably use `Date.toLocaleDateString()` here instead... or in future PR.

File: js/popup.js [hunk 5]
```diff
@@ -24,12 +24,30 @@ function showFeedbackPanel () {
   show('#feedback-panel')
 }
 
+// grabbed from http://stackoverflow.com/questions/13203518/javascript-date-suffix-formatting
+// for clean date formatting
+// TODO: find an alternate solution if we ever L10N
+function ordinal(date) {
```
Comment (pdehaan) [inline js/popup.js]: home/travis/build/mozilla/blok/js/popup.js:30:17: Missing space before function parentheses.

File: js/popup.js (hunk 4 above)
Comment (johngruen) [inline js/popup.js]: does that get us the 'th' in 20th?

File: html/popup.html (hunk 2 above)
Comment (johngruen) [inline html/popup.html]: yeah, i can fix this

Comment (johngruen) [inline html/popup.html]: Fixed, not sure why github is showing this outdated diff

File: html/popup.html [hunk 6]
```diff
@@ -51,35 +51,33 @@
...
+                <label for="images">Images</label><br>
+                <input class="breakage" type="radio" name="breakage" value="video" id="video">
+                <label for="video">Video</label><br>
+                <input class="breakage" type="radio" name="breakage" value="layout" id="layout">
+                <label for="layout">Layout</label><br>
+                <input class="breakage" type="radio" name="breakage" value="buttons" id="buttons">
+                <label for="buttons">Buttons</label><br>
+                <input class="breakage" type="radio" name="breakage" value="other" id="other">
+                <label for="other">Something else</label><br>
+              <div>
```
Comment (pdehaan) [inline html/popup.html]: Should be `</div>`

File: js/popup.js (hunk 4 above)
Comment (pdehaan) [inline js/popup.js]: It does not. But it gives you some basic L10N for "free". BEHOLD!

```js
const laborDay = new Date(2016, 8, 5, 6,4,3);
const opts = {weekday: 'long', month: 'long', day: 'numeric'};

console.log(laborDay.toLocaleDateString("en-US", opts)); // "Monday, September 5"

console.log(laborDay.toLocaleDateString("fr", opts)); // "lundi 5 septembre"
```

**Comment by pdehaan** [comment]
Apart from my stupid comments and a couple unclosed tags, everything seems fine.
Once we merge this, we should post the latest XPI on testpilot.dev and I can do some more testing. Plus I guess rename the experiment to "Tracking Protection" in Test Pilot since that seems to be the trend in this PR.

**Comment by groovecoder** [comment]
When all the nits are fixed, I'll create and sign an update `.xpi` and add it to this branch so it's available once this is merged.

**Comment by groovecoder** [comment]
Spot-check works well, and the new UI looks really nice! r+ after the nits are fixed.

**Comment by johngruen** [comment]
@pdehaan there's no such thing as stupid comments, only stupid commenters :)

**Comment by pdehaan** [comment]
> there's no such thing as stupid comments, only stupid commenters :)

**Comment by pdehaan** [comment]
> When all the nits are fixed, I'll create and sign an update .xpi and add it to this branch so it's available once this is merged.

r+. @groovecoder, your turn.

**Comment by groovecoder** [comment]
Works well and looks good!

**Comment by pdehaan** [comment]
> Works well and looks good!

Which is also _exactly_ what I wrote for @johngruen's endorsement on LinkedIn!

**Comment by groovecoder** [comment]
Built and signed 1.0rc1 `.xpi` for this branch and for @clouserw.

**Pull request merged**